src/assessments/seo/metaDescriptionKeywordAssessment.js

```diff
@@ -41,7 +41,7 @@ class MetaDescriptionKeywordAssessment extends Assessment {
 	 */
 	getResult( paper, researcher, i18n ) {
 		this._keywordMatches = researcher.getResearch( "metaDescriptionKeyword" );
-		var assessmentResult = new AssessmentResult();
+		let assessmentResult = new AssessmentResult();
 
 		assessmentResult.setScore( this.calculateScore() );
 		assessmentResult.setText( this.translateScore( i18n ) );
```

Comment:
For the sake of consistency with the other SEO assessments and also for convenience of editing, please define the feedback strings in the configuration instead of in the function `translateScore`.